WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -359,9 +359,9 @@ private extension SettingsViewController {
         navigationController?.pushViewController(betaFeaturesViewController, animated: true)
     }
 
-    func featureRequestWasPressed() {
-        let safariViewController = SFSafariViewController(url: WooConstants.URLs.featureRequest.asURL())
-        present(safariViewController, animated: true, completion: nil)
+    func presentSurveyForFeedback() {
+        let surveyNavigation = SurveyCoordinatingController(survey: .inAppFeedback)
```

Comment:
Looks like this lightway coordinator has already been amortized with it being called from 3 places already 😅

Reply:
Right!? It's so nice. ❤️